en/docs/learn/x509certificate-authenticator.md

```diff
@@ -0,0 +1,703 @@
+    ```
+
+    !!! tip "CN"
+        The User objects in the LDAP directory hierarchy have designators that start with CN, meaning Common Name. The CN designator applies to all but a few object types. Active Directory only uses two other object designators (although LDAP defines several).
+
+Once you have done the above steps, you have the keystore (`localcrt.jks`), truststore (`cacerts.jks`), and pkcs12 (`localhost.p12`) files that you need to use later on in this guide.
+
+## Configuring the X509 certificate for the app
+
+1.  Download the [WSO2 Identity Server](http://wso2.com/products/identity-server/).
+
+2.  Replace your keystore file path, keystore password, trust store file path and trust store password (you can use the keystore and
+    truststore, which you created in the [Working with Certificates](#working-with-certificates) section) in the following configuration and add it to the
+    `<IS_HOME>/repository/conf/deployment.toml` file.
+
+    ``` toml 
+    [custom_trasport.x509.properties]
+    protocols="HTTP/1.1"
+    port="8443"
+    maxThreads="200"
+    scheme="https"
+    secure=true
+    SSLEnabled=true
+    keystoreFile="/path/to/keystore.jks"
+    keystorePass="keystorepwd"
+    truststoreFile="/path/to/truststore.jks"
+    truststorePass="truststorespassword"
+    bindOnInit=false
+    clientAuth="want"
+    ssl_protocol = "TLS"
+    ```
+
+    !!! note
+    
+        1.   To function properly, this connector should come first in the order. Otherwise, when mutual SSL takes place, the already existing connector (9443) will be picked up and the certificate will not be retrieved correctly.
+
+        2.  The `clientAuth` attribute causes the Tomcat to require the client with providing a certificate that can be configured as follows.
+
+            -   `true` : valid client certificate required for a connection to succeed
+
+            -   `want` : use a certificate if available, but still connect if no certificate is available
+
+            -   `false` : no client certificate is required or validated
+    
+        -   The `truststoreFile` attributes specifies the location of the truststore that contains the trusted certificate issuers.
+    
+
+3.  Download the authenticator.jar file and the artifacts from the [WSO2 store](https://store.wso2.com/store/assets/isconnector/list?q=%22_default%22%3A%22X509%22).
+
+4.  Place the `           authenticator.jar          ` file in the
+    `           <IS_HOME>/repository/components/dropins          `
+    directory.  
+
+    !!! note
+        If you want to upgrade the X509 Certificate Authenticator in your
+        existing IS pack, refer [upgrade instructions.](https://docs.wso2.com/display/ISCONNECTORS/Authenticator+Upgrade+Instructions)
+    
+
+5.  Place the
+    `           x509certificateauthenticationendpoint.war          `
+    file in the
+    `           <IS_HOME>/repository/deployment/server/webapps          `
+    directory.
```

Comment:
These steps should not be required anymore since the authenticator is available in the IS pack by default. Please check on it